src/main/java/com/wa01/movies/rpfh/Main.java

```diff
@@ -0,0 +1,15 @@
+package com.wa01.movies.rpfh;
+
+public class Main {
```

Comment:
Since you are using unit tests now.
This class is no longer necessary.
Please remove it!